### DISCUSSION TITLE
How to get rid of this area

### DISCUSSION THREAD

**Discussion by stephandeckers** [valinet/ExplorerPatcher#3861] [Q&A] [answered]
![Capture](https://github.com/user-attachments/assets/0ae2fbe1-f86a-478d-a21a-b94617dd08d5)

Anybody knows how to get rid of this area ?

**Comment by pyrates999** [comment]
try this: https://answers.microsoft.com/en-us/windows/forum/all/remove-homegallery-from-explorer/28312555-d851-4794-83b1-60757b31f2c3

**Reply by stephandeckers** [reply]
I got to https://www.minitool.com/news/how-to-remove-gallery-from-file-explorer.html, but for me this information is outdated. This is how my registry looks like
![reg](https://github.com/user-attachments/assets/a8b37ae8-5dc3-41de-ba87-d882495fa73b)

**Reply by pyrates999** [reply, marked as answer]
Then I'm not sure.  I would try to google for your windows 11 version of 24h2 as that article seems to be for windows 11 23h2.

Since this involves modifying the registry, create a system restore point first so you can restore to it.

Perhaps they removed the registry key from NameSpace_41040327 to NameSpace, as I do see that registry key listed there under NameSpace instead of NameSpace_41040327.

Make a backup of the registry key before you modify the registry though.  Then reboot.  If it doesn't work, you can restore from the registry key backup you made.

And if it doesn't boot, restore from the system restore point that you made.

If you are not comfortable with doing any of this, I would leave this as is.
